src/core.e/LeastMost.pm6

```diff
@@ -0,0 +1,111 @@
+constant Least = Any.new;
```

Comment:
Why not use -Inf here? I get this is a stub, and the real value of $*LEAST is registered somewhere else. Still, it would make it a bit more consistent. Such as it is now, they are the same.